docs/modules/mssql.md

```diff
@@ -82,6 +82,68 @@ Drops the applicable user from the SQL Server.
 
 Deletes both the database and the SQL Server instance.
 
+##### Examples
+
+###### Kubernetes
+
+The `contrib/k8s/examples/sqldb/sqldb-all-in-one-instance.yaml` can be used to provision one of the plans from the all-in-one `azure-sql` service. This can be done with the following example:
+
+```console
+kubectl create -f ../../contrib/k8s/examples/sqldb/sqldb-all-in-one-instance.yaml
+```
+
+You can then create a binding to the service with the following command:
+
+```console
+kubectl create -f ../../contrib/k8s/examples/sqldb/sqldb-all-in-one-binding.yaml
+```
+
+###### Cloud Foundry
+
+Using the `cf` cli, you can create the `basic` plan of the `azure-sql` service with the following command:
+
+```console
+cf create-service azure-sql basic azure-sql-all-in-one -c '{
+        "resourceGroup" : "demo",
+        "location" : "eastus",
+        "firewallRules" : [
+            {
+                "name": "AllowAll",
+                "startIPAddress": "0.0.0.0",
+                "endIPAddress" : "255.255.255.255"
+            }
+        ]
+    }
+'
+```
+
+###### cURL
+
+Assuming your OSBA is running locally on port 8080 with the default username and password, you can provision the all-in-one Azure SQL Database service with a cURL command similar to the following example:
+
+```console
+curl -X PUT \
```

Comment:
You may want to make it more clear which values a user will need to replace, by setting them as env vars above the curl command, and then referencing them in this command.

Reply:
I will take care of this, along with https://github.com/Azure/open-service-broker-azure/issues/320 in a follow up PR.